common:
  frame_id: "lili_om"
  
preprocessing:
  surf_thres: 0.25
  edge_thres: 3

lidar_odometry: 
  max_num_iter: 15
  scan_match_cnt: 1 
  if_to_deskew: false 

backend_fusion: 
  imu_topic: "/imu/data"
  max_num_iter: 15
  local_map_width: 40 
  slide_window_width: 3 
  lidar_const: 25 
  reflect_thres: 15.0 
  surf_dist_thres: 0.08 
  kd_max_radius: 1.0 
  surf_ds: 0.4 
  edge_ds: 0.2 
  loop_closure_on: true
  lc_search_radius: 7 
  lc_map_width: 20 
  lc_icp_thres: 0.15 
  global_lc_time_thres: 60.0 
  local_lc_time_thres: 60.0 
  mapping_interval: 10
  save_pcd: false 
  mapping_ds: 0.3

#extrinsic imu to lidar
  ql2b_w: 0 
  ql2b_x: 0 
  ql2b_y: 0 
  ql2b_z: 1 

  tl2b_x: -0.0265 
  tl2b_y: 0.0202 
  tl2b_z: 0.05309 
